topics/metagenomics/tutorials/foodborne-pathogen-detection-nanopore-sequencing-data/tutorial.md

```diff
@@ -0,0 +1,1302 @@
+> {: .solution}
+{: .question}
+
+# Pathogen Tracking among all samples
+
+> <comment-title></comment-title>
+>
+> If you did not get your **Gene based pathogenic identification** section output files needed yet or you got an error with any, you can go on and upload them all or the ones missing from Zenodo so you can start this workflow, please don't forget to create the collections for them as explained in the pervious hands-on.
+>
+> > <hands-on-title>Optional Data upload</hands-on-title>
+> >
+> > 1. Import all tabular and FASTA files needed for this section via link from [Zenodo]({{ page.zenodo_link }}) to the new created history:
+> >
+> >    ```text
+> >    {{ page.zenodo_link }}/files/VFDB_Accession_Barcode10_Spike2.tabular
+> >    {{ page.zenodo_link }}/files/VFDB_Accession_Barcode11_Spike2b.tabular
+> >    {{ page.zenodo_link }}/files/VFDB_Accession_Tabular_with_SampleID_as_a_header_Barcode10_Spike2.tabular
+> >    {{ page.zenodo_link }}/files/VFDB_Accession_Tabular_with_SampleID_as_a_header_Barcode11_Spike2b.tabular
+> >    {{ page.zenodo_link }}/files/VFs_ABRicate_Sample_Specific_Formated_Tabular_Barcode10_Spike2.tabular
+> >    {{ page.zenodo_link }}/files/VFs_ABRicate_Sample_Specific_Formated_Tabular_Barcode11_Spike2b.tabular
+> >    {{ page.zenodo_link }}/files/Sample_Specific_Contigs_FASTA_file_Barcode10_Spike2.fasta
+> >    {{ page.zenodo_link }}/files/Sample_Specific_Contigs_FASTA_file_Barcode11_Spike2b.fasta
+> >    ```
+> >
+> >    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+> >
+> {: .hands_on}
+{: .comment}
+
+In this last section, we would like to show how to aggregate results and use the results to help tracking pathogenes among samples by:
+
+1. Drawing a presence-absence heatmap of the identified **VF** genes within all samples to visualize in which samples these genes can be found.
+2. Drawing a [phylogenetic tree](https://www.sciencedirect.com/topics/medicine-and-dentistry/phylogenetic-tree) for each pathogenic gene detected, where we will relate the contigs of the samples together where this gene is found.
+
+With these two types of visualizations we can have an overview of all samples and the genes, but also how samples are related to each other, which common pathogenic genes they share. Given the time of the sampling and the location one can easy identify using these graphes where and when the contamination has occurred among the different samples.
+
+> <hands-on-title>Organize data</hands-on-title>
+>
+> 1. Create a collection named `VFs` with `VFs` files for both tags
+>
+>    {% snippet faqs/galaxy/collections_build_list.md %}
+>
+> 2. Create a collection named `VFs accessions` with `VFs accessions` files for both tags
+>
+> 3. Create a collection named with `VFs accessions with SampleID` with `VFs accessions with SampleID as a header` files
```

Comment:
These files were not visible with the short version

Reply:
yeah these weren't their names, so shall I change the workflow or return back the names

Reply:
Please change in the workflow